docs/cli/mock-cmd.md

```diff
@@ -0,0 +1,140 @@
+# Mock your API locally using Kusk
+
+Spin up a local mocking server that generates responses from your content schema or returns your defined examples.
+
+Kusk uses [Docker](https://docs.docker.com/get-docker/) to launch a mock server container.
+
+All you need to get started is your OpenAPI definition.
+
+## Example
+### Provide your API
+Let's mock the following API using Kusk.
+
+```yaml
+openapi: 3.0.0
+info:
+  title: todo-backend-api
+  version: 0.0.2
+paths:
+  /todos:
+    get:
+      responses:
+        '200':
+          description: 'ToDos'
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  title:
+                    type: string
+                    description: Description of what to do
+                  completed:
+                    type: boolean
+                  order:
+                    type: integer
+                    format: int32
+                  url:
+                    type: string
+                    format: uri
+                required:
+                  - title
+                  - completed
+                  - order
+                  - url
+            application/xml:
+              example:
+                title: "Mocked XML title"
+                completed: true
+                order: 13
+                url: "http://mockedURL.com"
+            text/plain:
+              example: |
+                title: "Mocked Text title"
+                completed: true
+                order: 13
+                url: "http://mockedURL.com"
+```
+
+It has a single path with 3 content types.
+- `application/json` specifies a response schema which kusk will generate a generated response that matches
+- `applications/xml` and `text/plain` specifies examples which kusk will return as is.
+
+### Launch Kusk Mocking Server
+```shell
```

Comment:
I was thinking if we had a GIF here showing how it works. It would look very nice